api/EventType.json

```diff
@@ -0,0 +1,106 @@
+{
+  "api": {
+    "EventType": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Events",
+        "support": {
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "click": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Events/click",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
```

Comment:
`click` should have a sub-feature called `on_disabled_elements` containing the following:
```json
"on_disabled_elements": {
  "__compat": {
    "description": "On disabled form elements",
    "support": {
      "chrome": {
        "version_added": true,
        "partial_implementation": true,
        "notes": "Only works for <code>&lt;textarea&gt;</code> elements and some <code>&lt;input&gt;</code> element types."
      },
      "chrome_android": {
        "version_added": null
      },
      "edge": {
        "version_added": false
      },
      "edge_mobile": {
        "version_added": false
      },
      "firefox": {
        "version_added": false
      },
      "firefox_android": {
        "version_added": false
      },
      "ie": {
        "version_added": true,
        "partial_implementation": true,
        "notes": "Internet Explorer only triggers the <code>click</code> event on <code>&lt;input&gt;</code> elements of type <code>checkbox</code> or <code>radio</code> when the element is double-clicked."
      },
      "opera": {
        "version_added": true,
        "partial_implementation": true,
        "notes": "Only works for <code>&lt;textarea&gt;</code> elements and some <code>&lt;input&gt;</code> element types."
      },
      "opera_android": {
        "version_added": null
      },
      "safari": {
        "version_added": null
      },
      "safari_ios": {
        "version_added": null
      },
      "samsunginternet_android": {
        "version_added": null
      },
      "webview_android": {
        "version_added": null
      }
    },
    "status": {
      "experimental": false,
      "standard_track": true,
      "deprecated": false
    }
  }
}
```

Reply:
Why will I add this to the `click` event, is not sub-feature of `click`?

Reply:
It must be on the same level as the `__compat` block, but not inside the `__compat` block.

Also, [stuff like this](https://developer.mozilla.org/en-US/docs/Web/Events/click#Browser_compatibility) has been made into sub-features in the past: https://github.com/mdn/browser-compat-data/blob/5133d14b23996400d17fead58193e7c27c8d0f22/api/NavigatorConcurrentHardware.json#L52-L56 https://github.com/mdn/browser-compat-data/blob/0fd82d0c7e966a44ce89dd14f9e33fb00375041f/html/elements/html.json#L110-L114